pkgs/by-name/ca/carto/package.nix

```
@@ -19,6 +19,11 @@ buildNpmPackage rec {
 
   dontNpmBuild = true;
 
+  postInstall = ''
+    # Remove broken symlinks
+    find "$out" -xtype l -exec unlink {} \;
```

`find` can delete things on its own, no need for `unlink`.
I also suggest only searching for files in `$out/lib/node_modules` instead of in all of `$out`. I think this is better because we only expect dangling symlinks in this directory and if any other symlinks get broken outside of this directory in a future update, it would be nice to be alerted in this case.
```suggestion
    find "$out/lib/node_modules" -xtype l -delete
```

---

ohh that's cool I've always been used to -exec didn't really know about -delete, thanks for the suggestion I'm gonna make the update asap :)